Duffel.ApiClient/Duffel.ApiClient.csproj

```diff
@@ -6,14 +6,14 @@
         <Authors>Duffel Technology Ltd.</Authors>
         <Copyright>Duffel Technology Ltd.</Copyright>
         <Company>Duffel Technology Ltd.</Company>
-        <PackageVersion>0.1.0</PackageVersion>
+        <PackageVersion>0.2.0</PackageVersion>
         <Title>dotNet client for interacting with the Duffel API</Title>
         <PackageProjectUrl>https://duffel.com/docs/api/overview/welcome</PackageProjectUrl>
         <RepositoryUrl>https://github.com/duffelhq/duffel-api-dotnet</RepositoryUrl>
         <RepositoryType>Git</RepositoryType>
-        <AssemblyVersion>0.1.0</AssemblyVersion>
-        <FileVersion>0.1.0</FileVersion>
-        <PackageLicenseUrl>https://raw.githubusercontent.com/duffelhq/duffel-api-dotnet/main/LICENSE</PackageLicenseUrl>
+        <AssemblyVersion>0.2.0</AssemblyVersion>
+        <FileVersion>0.2.0</FileVersion>
+        <PackageLicenseUrl></PackageLicenseUrl>
```

Comment:
Setting licenseUrl here has been deprecated. Removing to await warning during build